vm-setup/roles/libvirt/templates/network.xml.j2

```diff
@@ -25,7 +25,7 @@
 {% set numflavor = lookup('vars', 'num_' + flavor + 's')|default(0)|int %}
 {% for num in range(0, numflavor) %}
 {% set ironic_name = ironic_prefix + flavor + "_" + num|string %}
-      <host mac='{{ node_mac_map.get(ironic_name).get(item.name)}}' name='{{flavor}}-{{num}}' ip='{{item.dhcp_range[0]|ipmath(ns.index|int)}}'/>
+      <host mac='{{ node_mac_map.get(ironic_name).get(baremetal).get(mac)}}' name='{{flavor}}-{{num}}' ip='{{node_mac_map.get(ironic_name).get(baremetal).get(ip)}}'/>
```

Comment:
don't hardcode the network name. and mac and ip are not variables, they are strings, hence need to be quoted